_scripts/captureLogs.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# captureLogs.sh is used to simplify the capture of all logs from a run
+# of tests. It takes the following arguments:
+#
+# captureLogs.sh dir command args...
+
+source "${BASH_SOURCE%/*}/common.bash"
+source "${BASH_SOURCE%/*}/tidyUp.bash"
+
+dir="$1"
```

Comment:
Nits, a usage dump if started w/o argument would be nice.

Reply:
Great suggestion; done.